obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  status: 1
+- agasc_id:
+  - 376724944
+  comments: Mag 6.5 spoiler 45 arcsec away
+  mp_starcat_time: 2015:331:14:10:26.457
+  status: 1
+- agasc_id:
+  - 195167088
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:355:17:05:45.600
+  status: 1
+- agasc_id:
+  - 615257632
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2016:021:00:11:54.978
+  status: 1
+- agasc_id:
+  - 643702120
+  comments: Sparse periods of track at 12.5 mag; set mag to MAXMAG
+  mp_starcat_time: 2016:027:20:27:48.432
+  status: 1
+- agasc_id:
+  - 417611952
+  comments: Bright star. Fails to track. Set mag to 5.0
+  mp_starcat_time: 2016:062:05:39:12.185
+  status: 1
+- agasc_id:
+  - 118096888
+  comments: Faint star. Magnitude is good
+  mp_starcat_time: 2016:127:20:16:33.087
+  status: 0
+- agasc_id:
+  - 458500128
+  comments: Unexplained variability
+  mp_starcat_time: 2016:136:08:42:06.470
+  status: 0
+- agasc_id:
+  - 118359864
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2016:141:00:48:34.306
+  status: 1
+- agasc_id:
+  - 397673760
+  comments: BSH at start of observation
+  mp_starcat_time: 2016:234:07:04:56.469
+  status: 1
+- agasc_id:
+  - 397674472
+  comments: BSH at start of observation
+  mp_starcat_time: 2016:234:07:04:56.469
+  status: 1
+- agasc_id:
+  - 397675248
+  comments: BSH at start of observation
+  mp_starcat_time: 2016:234:07:04:56.469
+  status: 1
+- agasc_id:
+  - 397679944
+  comments: BSH at start of observation
+  mp_starcat_time: 2016:234:07:04:56.469
+  status: 1
+- agasc_id:
+  - 397680904
+  comments: BSH at start of observation
+  mp_starcat_time: 2016:234:07:04:56.469
+  status: 1
+- agasc_id:
+  - 236326704
+  comments: ACA annealing
+  mp_starcat_time: 2016:257:09:35:53.151
+  status: 1
+- agasc_id:
+  - 236456008
+  comments: ACA annealing
+  mp_starcat_time: 2016:257:09:35:53.151
+  status: 1
+- agasc_id:
+  - 236470648
+  comments: ACA annealing
+  mp_starcat_time: 2016:257:09:35:53.151
+  status: 1
+- agasc_id:
+  - 615257632
+  comments: Almost never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2016:279:23:36:33.516
+  status: 1
+- agasc_id:
+  - 302263576
+  comments: UNKNOWN never acquired, but star has been observed at mag 9.3, MAXMAG=10.7
+  mp_starcat_time: 2016:310:08:24:22.999
+  status: 1
+- agasc_id:
+  - 314186320
+  comments: NSM at start of observation
+  mp_starcat_time: 2016:324:12:21:46.079
+  status: 1
+- agasc_id:
+  - 314189096
+  comments: NSM at start of observation
+  mp_starcat_time: 2016:324:12:21:46.079
+  status: 1
+- agasc_id:
+  - 314317160
+  comments: NSM at start of observation
+  mp_starcat_time: 2016:324:12:21:46.079
+  status: 1
+- agasc_id:
+  - 314318832
+  comments: NSM at start of observation
+  mp_starcat_time: 2016:324:12:21:46.079
+  status: 1
+- agasc_id:
+  - 314321208
+  comments: NSM at start of observation
+  mp_starcat_time: 2016:324:12:21:46.079
+  status: 1
+- agasc_id:
+  - 27919752
```

Comment:
Should also be added to bad list if the every SRCH got us back to the spoiler star (or we have bad position for star, either way).